### PR TITLE
Find all leap years occuring between any two years

### LINKED ISSUE #1365: Find all leap years between any two years in C, C++, Java, Python and Go
@marwahmanbir @rishabhgarg25699 Please assign it to me.

### PR DESCRIPTION
**Pull request by singh-shreya6**
Fixes #1365 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added implementation for finding all leap years between any two years 

#### Languages Used:
- C
- C++
- Java
- Python
- Go

#### Files Added:
- Leap_Year.c
- Leap_Year.cpp
- Leap_Year.java
- Leap_Year.py
- Leap_Year.go

### CODE REVIEW

File: Leap_Year/Leap_Year.c [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    if (year % 4 == 0)
+        return 1;
+
+    return 0;
+}
+
+int main()
+{
+    int year1 , year2;
```
Comment (marwahmanbir) [inline Leap_Year/Leap_Year.c]: Remove space before comma.

File: Leap_Year/Leap_Year.c [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    printf("Enter the first year:\n");
+    scanf("%d", &year1);
+    printf("Enter the second year year:\n");
+    scanf("%d", &year2);
+    printf("The leap years between %d and %d are: \n", year1, year2);
+    for(int i = year1; i <= year2; i++)
```
Comment (marwahmanbir) [inline Leap_Year/Leap_Year.c]: Space after for.

**Comment by singh-shreya6** [comment]
Done. please review @marwahmanbir .

**Comment by MastersAbh** [comment]
Please review this @marwahmanbir

**Pull request merged**